src/main/scala/org/apache/spark/deploy/armada/Config.scala

```diff
@@ -106,11 +93,7 @@ private[spark] object Config {
   private val validServiceNamePrefix: Regex = "([a-z][0-9a-z-]{0,29})?".r
 
   private[armada] def serviceNamePrefixValidator(name: CharSequence): Boolean = {
-    val serviceNameMaybe = validServiceNamePrefix.findPrefixMatchOf(label)
-    serviceNameMaybe match {
-      case Some(_) => true
-      case None => false
-    }
+    validServiceNamePrefix.findPrefixMatchOf(name).isDefined
```

Comment:
Note: this used to reference `label` and method argument `name` was unused.

Reply:
Ouch, good catch.  I missed it yesterday.  We need to be more serious about unit tests.